packages/tooling/fast-tooling-react/src/form/custom-controls/control.text-align.tsx

```diff
@@ -5,6 +5,12 @@ import styles, { TextAlignControlClassNameContract } from "./control.text-align.
 import { TextAlignControlProps } from "./control.text-align.props";
 import { classNames } from "@microsoft/fast-web-utilities";
 
+enum Direction {
+    Left = "left",
```

Comment:
should these be localized? Does this get rendered as text?

Reply:
doesn't look like it does, he has `this.props.strings.textAlignLeftLabel` for the translation.

Reply:
The enum values are not displayed and are used for comparison against values passed as a prop to the Text Align control. I added the enum to improve logic within the control but left it as a string comparison to the prop in order to keep the change internal to reduce risk of regression.